ctapipe/tools/stage1.py

```diff
@@ -623,6 +624,10 @@ def _setup_writer(self, writer: HDF5TableWriter):
                 writer.exclude(
                     f"/dl1/event/telescope/images/{table_name}", "image_mask"
                 )
+
+            writer.exclude(
+                f"/dl1/monitoring/telescope/trigger/{table_name}", "trigger_pixels"
+            )
```

Comment:
isn't this important for real data?

Reply:
At the DL1 level you want to know which trigger patches contributed to the trigger decision?

Reply:
Then we would need to convert it to a mask. Currently the list is not writable and this exclude is only to silence the warnings.